src/Toolkit/Toolkit.Maui/BookmarksView/BookmarksView.cs

```diff
@@ -67,17 +65,15 @@ protected override void OnApplyTemplate()
     {
         base.OnApplyTemplate();
 
-        if (_presentingView != null)
+        if (_presentingView is not null)
         {
-            _presentingView.ItemSelected -= Internal_bookmarkSelected;
+            _presentingView.SelectionChanged -= Internal_bookmarkSelected;
         }
 
-        _presentingView = GetTemplateChild("PresentingView") as ListView;
```

Comment:
Keep this line but update it for CollectionView. That ensures an old _presentingView value would get overwritten with null if it isn't found.